### PR TITLE
CORE-1870 Implement feedback from AH 

### PR DESCRIPTION
**Pull request by sboleyn**
Changes made include:

-  removing hour/min/sec from details and listing
- edit quota amounts in GiBs/ display quota and usage amounts in human readable format

### CODE REVIEW

File: src/components/subscriptions/edit/EditSubscription.js [hunk 1]
```diff
@@ -222,17 +222,9 @@ function EditSubscriptionForm(props) {
...
-                            parentId={buildID(
-                                parentId,
-                                ids.EDIT_SUB_DLG.USAGES
-                            )}
-                            {...arrayHelpers}
-                        />
-                    )}
+                <Usages
+                    parentId={buildID(parentId, ids.EDIT_SUB_DLG.USAGES)}
+                    usages={subscription.usages}
```
Comment (slr71) [inline src/components/subscriptions/edit/EditSubscription.js:227]: ✨

File: src/components/subscriptions/edit/Quotas.js [hunk 2]
```diff
@@ -69,17 +68,10 @@ function Quotas(props) {
...
-                                            ids.EDIT_QUOTAS_DLG
-                                                .QUOTAS_RESOURCE_TYPE
-                                        )}
-                                        fullWidth={false}
-                                        disabled
-                                        component={FormTextField}
+                                    <QuotaResourceType
+                                        quota={quota}
+                                        index={index}
+                                        t={t}
```
Comment (slr71) [inline src/components/subscriptions/edit/Quotas.js:74]: ✨

File: src/components/subscriptions/edit/Usages.js [hunk 3]
```diff
@@ -36,16 +37,7 @@ function Usages(props) {
...
-                                <TableCell>
-                                    <Typography>{resource.usage}</Typography>
-                                </TableCell>
-                                <TableCell>
-                                    <Typography>
-                                        {resource.resource_type.unit}
-                                    </Typography>
-                                </TableCell>
-                            </DERow>
+                            <UsageAmounts index={index} resource={resource} />
```
Comment (slr71) [inline src/components/subscriptions/edit/Usages.js:40]: ✨

**Review by slr71 — APPROVED** [review]
I didn't catch any problems. 👍

**Review by psarando — COMMENTED** [review]
LGTM 👍

**Pull request merged**